scripts/setup.sh

```
@@ -240,17 +255,32 @@ wait_for_resource deployment/automation-controller-operator-controller-manager c
+    echo "Deploying OSAC using Helm..."
+    helm dependency build charts/osac/
+    helm upgrade --install osac charts/osac/ \
+        --namespace "${INSTALLER_NAMESPACE}" \
+        --create-namespace \
+        --values "${VALUES_FILE}" \
+        --timeout 40m \
+        --wait
+else
+    # --- Kustomize deployment mode (legacy) ---
+    echo "Deploying OSAC using Kustomize..."
+    oc apply -k overlays/${INSTALLER_KUSTOMIZE_OVERLAY}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)